nettacker/database/db.py

```
@@ -70,17 +97,34 @@ def send_submit_query(session):
     Returns:
         True if submitted success otherwise False
     """
-    try:
-        for _ in range(1, 100):
+    if isinstance(session, tuple):
+        connection, cursor = session
```

Ruff (0.14.10)

Unpacked variable `cursor` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)